spec/controllers/selects_collections_controller_spec.rb

```diff
@@ -8,75 +10,74 @@ class SelectsCollectionsController < ApplicationController
 end
 
 describe SelectsCollectionsController, :type => :controller do
-  let(:user) { FactoryGirl.find_or_create(:jill) }
-  let(:dit) { FactoryGirl.find_or_create(:dit) }
+
+  let(:user){ FactoryGirl.create(:jill) }
+
   describe "Select Communities" do
-    before :each do
+    before :all do
       Collection.delete_all
       @community = Collection.new title: "Test Public Community" do |c|
-        c.apply_depositor_metadata(dit)
+        c.apply_depositor_metadata('user_one@example.com')
         c.is_community = true
         c.is_official = true
-        c.edit_users = [user.user_key, dit.user_key]
+        c.edit_users = ['user_two@example.com', 'user_one@example.com']
         c.save
       end
       @collection = Collection.new title: "Test Public Collection" do |c|
-        c.apply_depositor_metadata(dit)
-        c.edit_users =[user.user_key, dit.user_key]
+        c.apply_depositor_metadata('user_one@example.com')
+        c.edit_users = ['user_two@example.com', 'user_one@example.com']
         c.is_official = true
         c.save
       end
       @no_edit_community = Collection.new title: "Test No Edit Community" do |c|
-        c.apply_depositor_metadata(dit)
-        c.edit_users =[dit.user_key]
+        c.apply_depositor_metadata('user_one@example.com')
+        c.edit_users = ['user_one@example.com']
         c.is_community = true
         c.is_official = true
         c.is_admin_set = true
         c.save
       end
+    end
 
+
+    after :all do
+      Collection.delete_all
     end
 
+
     describe "Public Access" do
-      it "should return public communities" do
-        @user_communities = subject.find_communities
-        expect(@user_communities.map(&:id)).to match_array [@community.id, @no_edit_community.id]
-      end
-      it "should not return public collections" do
-        @user_communities = subject.find_communities
-        expect(@user_communities.map(&:id)).not_to include @collection.id
+      it "should return public communities and not public collections" do
+        expect(subject.find_communities.map(&:id)).to match_array [@community.id, @no_edit_community.id]
       end
     end
 
     describe "Regular User Read Access" do
```

Comment:
Previously the Read Access and Edit Access were the exact same? I got no idea what this code is doing but I assume its not testing the right things. Just passing in `:read` and `:edit` respectfully and all seems to still pass?